webpack/assets/javascripts/react_app/components/common/forms/validators.js

```diff
@@ -0,0 +1,17 @@
+import { translate as __, sprintf } from '../../../../react_app/common/I18n';
+
+export const maxLengthMsg = number => [
+  number,
+  sprintf(__('is too long (maximum is %s characters)'), number),
+];
+
+export const requiredMsg = () => __("can't be blank");
+
+export const errorsToSentence = (...validators) => value => {
```

Comment:
would this combine the api errors as well (e.g. name is already been taken?)

Reply:
The original thought was to combine error messages from validators that were applied to a particular form field. It is no longer needed because we use schema validation instead, removing.